### DISCUSSION TITLE
v-navigation-drawer

### DISCUSSION THREAD

**Discussion by Atefeh-Mirzabeigi** [vuetifyjs/vuetify#12468] [Q&A]
clipped in v-navigation-drawer doesn't work. (clipped-left, clipped-right in app-bar)

**Comment by jacekkarczmarczyk** [comment]
That's not actionable. If you're reporting a bug please use <https://issues.vuetifyjs.com/> and provide a reproduction. If you're having problem with your code we're also not able to help without seeing the code.
